include/print_utils.h

```diff
@@ -193,6 +213,21 @@ struct as_sub_second {
 	std::chrono::duration<double> seconds;
 };
 
+/// Wrap a byte count in this to auto-format it as KiB / MiB / etc.
+struct as_binary_size {
+	as_binary_size(double bytes) : bytes(bytes) {}
+	as_binary_size(size_t bytes) : bytes(static_cast<double>(bytes)) {}
+	double bytes;
+};
+
+/// Wrap a byte-per-second ratio in this to auto-format it as KiB/s, MiB/s, ...
```

Comment:
Bikeshedding: do we want power-of-2 units for transfer speeds? HW capabilities are usually specified with metric prefixes.

Reply:
Done.